package.json

```diff
@@ -48,7 +48,6 @@
     "got": "^9.6.0",
     "i18next": "^17.0.13",
     "jsonfile": "^5.0.0",
-    "mz": "^2.7.0",
```

Comment:
this package outdated and there is native fs which is similar and better to use

Reply:
native fs is not Promise aware right?  So you can't `await` on anything in `fs`